### ISSUE TITLE
Deprecation notices on PHP 8.1

### ISSUE THREAD

**Issue by swissspidy** [google/site-kit-wp#7497]
## Bug Description

```
[09-Aug-2023 03:14:45 UTC] PHP Deprecated:  stripos(): Passing null to parameter #1 ($haystack) of type string is deprecated in /wp-content/plugins/google-site-kit/includes/Core/REST_API/REST_Routes.php on line 69
[09-Aug-2023 03:14:53 UTC] PHP Deprecated:  stripos(): Passing null to parameter #1 ($haystack) of type string is deprecated in /wp-content/plugins/google-site-kit/includes/Core/REST_API/REST_Routes.php on line 69
```

## Steps to reproduce

N/A

## Additional Context

 - PHP Version:  8.1
 - Browser: [e.g. chrome, safari]
 - Plugin Version: 1.106.0

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by bethanylang** [comment]
Thanks, @swissspidy! I think this is a duplicate of #7147 – can you confirm? We'll be pushing that fix in our 1.108.0 release next week.

**Comment by swissspidy** [comment]
Ah yes looks like it. Sweet!
